backend/src/api/stats.ts

```diff
@@ -31,13 +38,20 @@ interface Stats {
 }
```

Comment:
isGlobalViewAdmin and getOrgMemberships are no longer being used, these should be removed.

Reply:
Resolved e578861fa6d61912c7f0d44cb8e69de7011e51d0